src/Compilers/CSharp/Test/IOperation/IOperation/IOperationTests_IUsingStatement.cs

```diff
@@ -5855,7 +6212,7 @@ void M(bool b)
         Block[B3] - Block
             Predecessors (0)
             Statements (1)
-                IInvocationOperation (virtual void P.Dispose()) (OperationKind.Invocation, Type: System.Void, IsImplicit) (Syntax: 'x = new P()')
```

Comment:
>virtual [](start = 38, length = 7)

Do we have at least one test with a pattern-based disposal utilizing a virtual method?

Reply:
Added UsingDeclaration_Flow_23 and 24

---
In reply to: [539514783](https://github.com/dotnet/roslyn/pull/49842#discussion_r539514783) [](ancestors = 539514783)